moztrap/model/library/api.py

```diff
@@ -450,8 +515,8 @@ def dehydrate(self, bundle):
         bundle.data["case_id"] = unicode(case.id)
         bundle.data["productversion_name"] = bundle.obj.productversion.name
         bundle.data["priority"] = unicode(case.priority)
-        bundle.data["created_by"] = unicode(case.created_by)
-        bundle.data["modified_by"] = unicode(case.modified_by)
-        bundle.data["modified_on"] = unicode(case.modified_on)
+        # bundle.data["created_by"] = unicode(case.created_by)
```

Comment:
are these to be removed?  We should just remove rather than commenting them out.